test/hotspot/jtreg/runtime/cds/appcds/loaderConstraints/LoaderConstraintsTest.java

```diff
@@ -34,6 +33,20 @@
  * @run driver LoaderConstraintsTest
  */
 
+/**
+ * @test id=custom-cl
+ * @requires vm.cds.custom.loaders
+ * @summary Test class loader constraint checks for archived classes with custom class loader
+ * @bug 8267347
```

Comment:
You've omitted the 8267754 bug id. Please also add the current bug id.

Reply:
I think the bug IDs should only include the product bugs that we are testing to test. The current bug (8267958) and 8267754 are both bugs in this test itself, so including them here isn't very useful.